.github/actions/image-scan/action.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Scan docker image'
+description: Scan docker image with trivy and dockle
+inputs:
+  image:
+    description: 'Image file to scan'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Run Dockle
+      uses: erzz/dockle-action@v1
+      with:
+        image: ${REGISTRY}/backbase/${GITHUB_REPOSITORY##*/}:${{ inputs.version }}
+        report-format: sarif
+        report-name: dockle-report
+        failure-threshold: fatal
+        exit-code: 1
+        dockle-version: 0.3.16
+        accept-extensions: pem
+
+    - name: Upload docle scan results to GitHub Security tab
+      uses: github/codeql-action/upload-sarif@v1.2.0
+      with:
+        sarif_file: dockle-report.sarif
+
+    - name: Run Trivy vulnerability scanner
+      uses: aquasecurity/trivy-action@master
+      with:
+        image-ref: '${REGISTRY}/backbase/${GITHUB_REPOSITORY##*/}:${{ inputs.version }}'
```

Comment:
input seems to be `${REGISTRY}/backbase/${GITHUB_REPOSITORY##*/}:${{ inputs.version }}`